frontend/src/App.js

```diff
@@ -15,6 +15,7 @@ import EmailConfirmPage from "./pages/EmailConfirmPage";
 import RequestPasswordResetPage from "./pages/RequestPasswordResetPage";
```

Comment:
As you edited this file, I think you should add yourself to the copyright at the top of the file.